catalog/resource_sql_table_test.go

```diff
@@ -33,6 +33,36 @@ func TestResourceSqlTableCreateStatement_External(t *testing.T) {
 	assert.Contains(t, stmt, "COMMENT 'terraform managed'")
 }
 
+func TestResourceSqlTableCreateStatement_IdentityColumn(t *testing.T) {
+	ti := &SqlTableInfo{
+		Name:                  "bar",
+		CatalogName:           "main",
+		SchemaName:            "foo",
+		TableType:             "EXTERNAL",
+		DataSourceFormat:      "DELTA",
+		StorageLocation:       "s3://ext-main/foo/bar1",
+		StorageCredentialName: "somecred",
+		Comment:               "terraform managed",
+		ColumnInfos: []SqlColumnInfo{
+			{
+				Name:     "id",
+				Type:     "bigint",
+				Identity: "default",
+			},
+			{
+				Name:    "name",
+				Comment: "a comment",
+			},
+		},
+	}
+	stmt := ti.buildTableCreateStatement()
+	assert.Contains(t, stmt, "CREATE EXTERNAL TABLE `main`.`foo`.`bar`")
+	assert.Contains(t, stmt, "USING DELTA")
+	assert.Contains(t, stmt, "(`id` BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL, `name`  NOT NULL COMMENT 'a comment')")
+	assert.Contains(t, stmt, "LOCATION 's3://ext-main/foo/bar1' WITH (CREDENTIAL `somecred`)")
+	assert.Contains(t, stmt, "COMMENT 'terraform managed'")
+}
```

Comment:
Can you also add an integration test to this file?
https://github.com/databricks/terraform-provider-databricks/blob/main/internal/acceptance/sql_table_test.go#L13

Reply:
Done

Reply:
Sorry for the missunderstanding: I was talking to add a test in the linked file so we can test this against the a real workspace.

Reply:
My bad, sorry. Will add.

Reply:
I have added an integration test. However, I do not know how to run it.

Reply:
I did run the tests and they succeed. Thanks!

Reply:
Wohooo! Would you still like me to test locally or is this good enough? I have trouble of setting up what Tanmay suggested.

Reply:
No needed to.

Reply:
No need to since tests are passing. Thank!